### ISSUE TITLE
On initial UI AppImage startup: { "name": "ContextError" }

### ISSUE THREAD

**Issue by schlegel11** [loft-sh/devpod#1035]
**What happened?**
On initial DevPod app image startup so after:

- `chmod +x ./DevPod_linux_amd64.AppImage`
- `./DevPod_linux_amd64.AppImage`

The UI opens with error pane (see screenshot) `{ "name": "ContextError" }`.
![devpod-bug](https://github.com/loft-sh/devpod/assets/5841076/5abdd56a-b9b3-43a7-bf55-396b529dffe7)

Further the DevPod.log contents:
```
[2024-04-28][17:15:19][INFO][devpod_desktop] Setup application
[2024-04-28][17:15:19][WARN][tauri_plugin_store] Failed to load store "/home/user/.local/share/sh.loft.devpod/.providers.json" from disk: File or directory not found (os error 2). Falling back to default values.
[2024-04-28][17:15:20][INFO][devpod_desktop] Setup done
[2024-04-28][17:15:20][INFO][devpod_desktop] Run
[2024-04-28][17:15:20][INFO][devpod_desktop::server] Listening on 127.0.0.1:25842
[2024-04-28][17:15:21][WARN][tauri_plugin_store] Failed to load store ".settings.json" from disk: File or directory not found (os error 2). Falling back to default values.
[2024-04-28][17:15:21][WARN][tauri_plugin_store] Failed to load store ".providers.json" from disk: File or directory not found (os error 2). Falling back to default values.
``` 

It just happens on initial startup. Afterwards there is no such behaviour and everything works fine.

**What did you expect to happen instead?**  

No error screen on initial startup. Just normal overview/main pane opens.

**How can we reproduce the bug?** (as minimally and precisely as possible)  

Start the app image for the first time like:

- `chmod +x ./DevPod_linux_amd64.AppImage`
- `./DevPod_linux_amd64.AppImage`

**Local Environment:**  
- DevPod Version: 0.5.5
- Operating System: linux
- ARCH of the OS: AMD64

**Comment by pascalbreuninger** [comment]
Hey @schlegel11, thanks for opening the issue, we'll look into it

**Comment by schlegel11** [comment]
Maybe there is no need to investigate this issue :wink: 
It seems to be fixed with version 0.5.6. At least for me.

**Comment by pascalbreuninger** [comment]
perfect! I'll go ahead and close this issue. If you experience this error again let us know
